src/hooks/useSelectionBounds.tsx

```diff
@@ -12,10 +12,13 @@ export function useSelectionBounds(): {
 } {
   const [selectionPosition, setSelectionPosition] =
     useState<SelectionBounds | null>(null);
+
   useEffect(() => {
     const onEnd = (e: MouseEvent) => {
       const selection = document.getSelection();
-      if (selection && !selection.isCollapsed) {
+      if (e.target instanceof Node && document.getElementById("demo-root")!.contains(e.target)){
```

Comment:
@andymatuschak FYI - I had to make this change but let me know if it feels off

Reply:
That seems like a fine approach for now!